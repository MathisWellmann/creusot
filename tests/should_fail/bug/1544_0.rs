extern crate creusot_contracts;

// Should emit a readable error message, not a crash
pub fn unsupported_type(_x: &dyn std::fmt::Debug) {}
