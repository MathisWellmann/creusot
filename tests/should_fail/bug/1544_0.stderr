error: Unsupported type: dyn std::fmt::Debug
 --> 1544_0.rs:4:25
  |
4 | pub fn unsupported_type(_x: &dyn std::fmt::Debug) {}
  |                         ^^

error: aborting due to 1 previous error

